src/services/npm-client/npm-client.ts

```diff
@@ -0,0 +1,7 @@
+import { executeCommand } from '../child-process-client';
+
+export const packPackage = async (packagePath: string): Promise<string> => {
+  const packOutput = await executeCommand('npm pack', { cwd: packagePath });
```

Comment:
We already talked about this - we don't know what needs to be packaged. But with npm publishing story - we will need to pack in a different step anyways, so this will be changed.

Reply:
We know, it is `dist` folder for now. No packages.json, no types, not anything else, but `dist`.